From 60545c63716ecc720728c221c61d575b267fbfc8 Mon Sep 17 00:00:00 2001
From: Michal Schmidt <mschmidt@redhat.com>
Date: Tue, 21 Jul 2015 15:51:16 +0200
Subject: [PATCH] core: remove generic job completion messages from unit
 vtables

These units' message format strings are identical to the generic
strings. Since we can always rely on the fallback, these are now
redundant.

(cherry picked from commit c382d69e3d39daedebcedb2da882beeb147a3cda)

Related: #1506256
---
 src/core/automount.c | 1 -
 src/core/busname.c   | 3 ---
 src/core/mount.c     | 1 -
 src/core/service.c   | 3 ---
 src/core/slice.c     | 1 -
 src/core/socket.c    | 1 -
 src/core/swap.c      | 1 -
 src/core/target.c    | 1 -
 8 files changed, 12 deletions(-)

diff --git a/src/core/automount.c b/src/core/automount.c
index 679fe071e..08519e49c 100644
--- a/src/core/automount.c
+++ b/src/core/automount.c
@@ -1126,7 +1126,6 @@ const UnitVTable automount_vtable = {
                 .finished_start_job = {
                         [JOB_DONE]       = "Set up automount %s.",
                         [JOB_FAILED]     = "Failed to set up automount %s.",
-                        [JOB_DEPENDENCY] = "Dependency failed for %s.",
                 },
                 .finished_stop_job = {
                         [JOB_DONE]       = "Unset automount %s.",
diff --git a/src/core/busname.c b/src/core/busname.c
index f626ba96d..a5e659049 100644
--- a/src/core/busname.c
+++ b/src/core/busname.c
@@ -1064,13 +1064,10 @@ const UnitVTable busname_vtable = {
                 .finished_start_job = {
                         [JOB_DONE]       = "Listening on %s.",
                         [JOB_FAILED]     = "Failed to listen on %s.",
-                        [JOB_DEPENDENCY] = "Dependency failed for %s.",
-                        [JOB_TIMEOUT]    = "Timed out starting %s.",
                 },
                 .finished_stop_job = {
                         [JOB_DONE]       = "Closed %s.",
                         [JOB_FAILED]     = "Failed stopping %s.",
-                        [JOB_TIMEOUT]    = "Timed out stopping %s.",
                 },
         },
 };
diff --git a/src/core/mount.c b/src/core/mount.c
index f726d9659..0dc67dde6 100644
--- a/src/core/mount.c
+++ b/src/core/mount.c
@@ -1978,7 +1978,6 @@ const UnitVTable mount_vtable = {
                 .finished_start_job = {
                         [JOB_DONE]       = "Mounted %s.",
                         [JOB_FAILED]     = "Failed to mount %s.",
-                        [JOB_DEPENDENCY] = "Dependency failed for %s.",
                         [JOB_TIMEOUT]    = "Timed out mounting %s.",
                 },
                 .finished_stop_job = {
diff --git a/src/core/service.c b/src/core/service.c
index 9622ce11f..8303a1e7e 100644
--- a/src/core/service.c
+++ b/src/core/service.c
@@ -3398,13 +3398,10 @@ const UnitVTable service_vtable = {
                 .finished_start_job = {
                         [JOB_DONE]       = "Started %s.",
                         [JOB_FAILED]     = "Failed to start %s.",
-                        [JOB_DEPENDENCY] = "Dependency failed for %s.",
-                        [JOB_TIMEOUT]    = "Timed out starting %s.",
                 },
                 .finished_stop_job = {
                         [JOB_DONE]       = "Stopped %s.",
                         [JOB_FAILED]     = "Stopped (with error) %s.",
-                        [JOB_TIMEOUT]    = "Timed out stopping %s.",
                 },
         },
 };
diff --git a/src/core/slice.c b/src/core/slice.c
index 9154558b7..1cce3e121 100644
--- a/src/core/slice.c
+++ b/src/core/slice.c
@@ -299,7 +299,6 @@ const UnitVTable slice_vtable = {
         .status_message_formats = {
                 .finished_start_job = {
                         [JOB_DONE]       = "Created slice %s.",
-                        [JOB_DEPENDENCY] = "Dependency failed for %s.",
                 },
                 .finished_stop_job = {
                         [JOB_DONE]       = "Removed slice %s.",
diff --git a/src/core/socket.c b/src/core/socket.c
index 771af0d24..efefe7ce5 100644
--- a/src/core/socket.c
+++ b/src/core/socket.c
@@ -2736,7 +2736,6 @@ const UnitVTable socket_vtable = {
                 .finished_start_job = {
                         [JOB_DONE]       = "Listening on %s.",
                         [JOB_FAILED]     = "Failed to listen on %s.",
-                        [JOB_DEPENDENCY] = "Dependency failed for %s.",
                         [JOB_TIMEOUT]    = "Timed out starting %s.",
                 },
                 .finished_stop_job = {
diff --git a/src/core/swap.c b/src/core/swap.c
index 984be2d9a..e71de4e65 100644
--- a/src/core/swap.c
+++ b/src/core/swap.c
@@ -1525,7 +1525,6 @@ const UnitVTable swap_vtable = {
                 .finished_start_job = {
                         [JOB_DONE]       = "Activated swap %s.",
                         [JOB_FAILED]     = "Failed to activate swap %s.",
-                        [JOB_DEPENDENCY] = "Dependency failed for %s.",
                         [JOB_TIMEOUT]    = "Timed out activating swap %s.",
                 },
                 .finished_stop_job = {
diff --git a/src/core/target.c b/src/core/target.c
index 2411a8e75..45248ad02 100644
--- a/src/core/target.c
+++ b/src/core/target.c
@@ -231,7 +231,6 @@ const UnitVTable target_vtable = {
         .status_message_formats = {
                 .finished_start_job = {
                         [JOB_DONE]       = "Reached target %s.",
-                        [JOB_DEPENDENCY] = "Dependency failed for %s.",
                 },
                 .finished_stop_job = {
                         [JOB_DONE]       = "Stopped target %s.",
